test-cases/gutenberg/color-settings.md

```diff
@@ -0,0 +1,183 @@
+
+# Color Settings - Test Cases
+
+#### **Precondition**
+
+Added block with color support such as `Buttons` or `Cover` 
+
+--------------------------------------------------------------------------------
+
+##### TC001
+
+### Colors segmented controls
+
+**Steps:**
+
+* Press `Background Color` / `Color`
+* Switch several times between `Solid` and `Gradient` segments 
+
+**Expected behavior:**
+
+* Selection animation works smoothly
+* Color swatches fade parallel with active segment change
```

Comment:
it is not very obvious what smoothly means, could you add a gif to decribe?

Reply:
done!